package.json

```diff
@@ -7,6 +7,9 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "watch": "nodemon index.js",
     "seed": "node seeds/seed.js",
+    "seed_heroku": "dotenv -e .env.jaws -- node seeds/seed.js",
+    "syncdb_local": "node db/index.js",
```

Comment:
How does this work?

Reply:
force: true is isolated to a separate file to restart the db in a controlled way